resources/lib/sony.py

```diff
@@ -372,13 +372,13 @@ def put_resume_time(self, airing_id, channel_id, program_id, series_id, tms_id):
         payload += '"tms_id":"'+tms_id+'",'
         payload += '"airing_id":'+airing_id+','
         payload += '"last_watch_date":"2017-11-13T06:54:419Z",'
-        payload += '"last_timecode":"00:00:00",'
+        payload += '"last_timecode":'+'"'+ResTime+'"'+','
         payload += '"start_timecode":"00:00:00:00",'
         payload += '"fully_watched":false,'
         payload += '"stream_type":"null"}'
 
         #xbmc.log("The Payload is: " + payload)
-        requests.put(url, headers=headers, data=payload, verify=self.verify)
+        r = requests.put(url, headers=headers, data=payload, verify=self.verify)

```

Comment:
The r = is superfluous here. Unless you need to evaluate the results no need to create a variable.